modules/test/playwright/tests/portal-workflow-kaleo-designer-web/utils/postSingleApproverCopy.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ApiHelpers} from '../../../helpers/ApiHelpers';
+import {getRandomInt} from '../../../utils/getRandomInt';
+import getRandomString from '../../../utils/getRandomString';
+
+export default async function postSingleApproverCopy(apiHelpers: ApiHelpers) {
+	const singleApproverWorkflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.getWorkflowDefinitionByName(
+			'Single Approver'
+		);
+
+	const workflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
+			'Copy of Single Approver' + getRandomInt(),
+			{
+				...singleApproverWorkflowDefinition,
+				externalReferenceCode: getRandomString(),
+			}
+		);
+
+	return workflowDefinition;
+}
```

Comment:
hey @igor-franca, can you include a blank line at the end of the file?

Reply:
We already have this extra line when we view the file, I think by default git is not displaying it or vsCode is responsible for showing it to us, I don't know for sure.

But when trying to add one more extra line at the end of the file, it ends up being removed by sf, so I think it's ok 😄

Reply:
nevermind this, Igor showed me that this last line is present at the file. For some reason git doesn't show it for any file, but all's good!